VERSIONING.md

```diff
@@ -0,0 +1,70 @@
+# Versioning
+
+This project is deployed in snapshot and release modes. Both have their own versioning strategy.
```

Comment:
I'll swap snapshot and release to be consistent with the order in which you explain it later
Maybe adding links too to the relevant paragraphs ?